src/test/java/com/mongodb/kafka/connect/sink/StartedMongoSinkTaskTest.java

```diff
@@ -189,35 +212,90 @@ void putTolerateAllOrderedWriteError() {
             asList(
                 // batch1
                 Records.simpleValid(TEST_TOPIC, 0),
+                Records.simpleValid(TEST_TOPIC, 1),
```

Comment:
Changed the first batch, which is successful, such that it contains multiple records. As a result, indices in all other batches were shifted.